joy : 
  ButtonA: 2 #O
  ButtonB: 3 #X
  ButtonX: 1 #Δ
  ButtonY: 0 #□
  ButtonLB: 4
  ButtonRB: 6
  ButtonSelect: 8 #share
  ButtonOption : 9 
  ButtonStart: 10 #logo
  ButtonTouchpad : 11
  ButtonLeftThumb: 12
  ButtonRightThumb: 13
  ButtonLeft : 14
  ButtonForward : 15
  ButtonRight : 16
  ButtonBackward : 17
  AxisLeftThumbX: 0
  AxisLeftThumbY: 1
  AxisRightThumbX: 2
  AxisRightThumbY: 3
  AxisLeftTrigger: 4
  AxisRightTrigger: 5